src/components/ModalPage/ModalPage.tsx

```diff
@@ -88,8 +98,11 @@ const ModalPageComponent = ({
   ]);
 
   const isDesktop = useAdaptivityIsDesktop();
-  const canShowCloseBtn =
-    viewWidth >= ViewWidth.SMALL_TABLET || platform === Platform.VKCOM;
+  const isCloseButtonShown =
```

Comment:
```suggestion
  // TODO v5.0.0 поправить под новую адаптивность
  const isCloseButtonShown =
```

чтобы при мерже не пропустили